### PR TITLE
Update installdefaultdatasettings

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update installdefaultdatasettings: 

- [x] change 'data type' to 'Data Type'

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco CMS v10, 12 & 13

### CODE REVIEW

File: 10/umbraco-cms/reference/configuration/installdefaultdatasettings.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ When Umbraco is installed for the first time, it creates a set of default data.
 
 In certain setups, you may want to take control over what is installed and opt-out of the creation of certain items.
 
-When working in a team and using a tool like Umbraco Deploy for sharing schema updates, when a colleague first sets up the project in their local environment, having this default data installed may not always be useful.
+When working in a team and using Umbraco Deploy for schema updates, consider the colleague's local project setup. The default data may not always be useful.
 
-For example, if a different set of languages from Umbraco's default (en-US) have been set up, it would be preferable to not have the default recreated. In other situations, certain Umbraco default Data, Member and Media Types may not be required.
+For example, if different languages are set up in Umbraco, it's better not to recreate the default language (en-US). In other situations, certain Umbraco default Data, Member and Media Types may not be required.
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/configuration/installdefaultdatasettings.md:18]: as it mentions in the original sentence it mentions that if "a different set of languages **from** Umbraco's default (en-US) have been set up":
```suggestion
For example, if different languages are set up in Umbraco, it's better not to recreate them from the default language (en-US). In other situations, certain Umbraco default Data, Member and Media Types may not be required.
```

File: 10/umbraco-cms/reference/configuration/installdefaultdatasettings.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ When Umbraco is installed for the first time, it creates a set of default data.
 
 In certain setups, you may want to take control over what is installed and opt-out of the creation of certain items.
 
-When working in a team and using a tool like Umbraco Deploy for sharing schema updates, when a colleague first sets up the project in their local environment, having this default data installed may not always be useful.
+When working in a team and using Umbraco Deploy for schema updates, consider the colleague's local project setup. The default data may not always be useful.
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/configuration/installdefaultdatasettings.md:16]:
```suggestion
When working in a team and using Umbraco Deploy for schema updates, consider your colleague's local project setup. The default installed data may not always be useful.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR 😊 

I have just few suggestions, can you please look into them and let me know what you think? 
(and if you agree with them, please make the changes to the other versions as well 💪 )

**Comment by erikjanwestendorp** [comment]
@alina-tincas Thanks for your suggestions. Just updated the PR 😄

**Review by alina-tincas — APPROVED** [review]

**Pull request merged**